Terminal.Gui/Views/TextView.cs

```diff
@@ -670,6 +690,8 @@ public override bool ProcessKey (KeyEvent kb)
 			case Key.CursorDown:
 			case Key.ControlP:
 			case Key.CursorUp:
+			case Key.PageUp:
+			case Key.PageDown:
 				lastWasKill = false;
```

Comment:
Other aliases include Alt-v for for page up, and control-v for page down

Reply:
I will add those too.